html/tuttifrutti/resources/images/ico_donate_2.png

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```

Comment:
Not a png?

Reply:
Item removed + section removed from index.html